### PR TITLE
Remove peer.exchanged events + patches

### PR DESCRIPTION
**Pull request by chmanie**
This PR removes the flaky `peer.exchanged` events and replaces them with... pretty much nothing. In the tests we query the store a couple of times until we know that it is synced.

This effectively also removes the patches and the scripts which apply them

### CODE REVIEW

File: src/StoreManager.ts [hunk 1]
```diff
@@ -70,27 +70,10 @@ class StoreManager {
...
-    ): void => {
-      // @todo The `peer` argument sometimes is undefined. It might not be a big
-      // problem as the replication works properly. But we should keep an eye on
-      // it.
-      log(
-        `Store "${address}" replicated for ${peer}. Got ${
-          heads.length
-        } new heads.`,
-      );
-      this.events.emit('stores:replicated', { address, heads, peer });
```
Comment (thiagodelgado111) [inline src/StoreManager.ts]: So this will be totally fire and forget from the client-side? No hook to store `replicated`?

Comment (thiagodelgado111) [reply src/StoreManager.ts]: Is it as flaky as depending on `peer.exchanged`?

Comment (chmanie) [reply src/StoreManager.ts]: Yes, but it was like that before. We didn’t use the event in the client, only in the tests

File: src/__tests__/integration.test.ts [hunk 2]
```diff
@@ -97,6 +97,38 @@ const createKVStore = async (
...
+    const interval = setInterval(
+      () => publishMessage(ipfs, room, action),
+      2000,
+    );
+    const timeout = setTimeout(reject, 10000);
+    const handleMessage = (msg: IPFS.PubsubMessage) => {
+      const {
+        type,
+        payload: { address, count },
+      } = JSON.parse(msg.data.toString());
```
Comment (JamesLefrere) [inline src/__tests__/integration.test.ts]: Does this perhaps need a try/catch and `reject`?

**Pull request merged**